test/app.js

```diff
@@ -1,8 +1,7 @@
 var request = require('supertest');
 var assert = require('assert');
 
-var Habitat = require('habitat');
-var testEnv = new Habitat();
+var testEnv = require('../src/environments');
```

Comment:
I tried the change out for you -- you can merge the changes to your branch by going here:
https://github.com/aid29/PaperBadger/pull/1

And clicking 'merge'